lps-client-backend/src/main/kotlin/no/nav/helsearbeidsgiver/lps/LpsSchema.kt

```diff
@@ -53,26 +53,17 @@ data class Periode(
     val tom: LocalDate,
 )
 
-@Serializable
-data class InnsendtInntektsmelding(
-    val dokument: JsonObject,
-    val orgnr: String,
-    val fnr: String,
-    val foresporsel_id: String?,
-    val innsendt_tid: String,
-    val mottatt_tid: String,
-)
-
 @Serializable
 data class InntektsmeldingRequest(
```

Comment:
HentInntektsmeldingRequest?

Reply:
Er det forresten dokumentert hvordan disse feltene påvirker hverandre? F. eks. hvis jeg bruker `fnr` og `navReferanseId` samtidig.

Reply:
Dette er kun for intern testing / POC, så ingen dokumentasjon her. Men parametere blir AND'et, for the record